tests/ui/platform_selector.js

```diff
@@ -0,0 +1,270 @@
+/*
+    Will replaces compat_filter.js tests alongside release of new navigation.
+    Platform selector manually rendered while settings.mktNavEnabled flag is
+    default to off.
+    Many tests skipped by Phantom since its HTMLSelectElement prototype is
+    not to spec, preventing the set up of <mkt-select>.
+*/
+var appList = helpers.load('app_list');
+var constants = helpers.load('constants');
+var jsuri = helpers.npm('jsuri/Uri');
+
+
+function phantomSkip(test) {
+    if (helpers.browser.isPhantom) {
```

Comment:
It might just be a PhantomJS quirk. Newer Webkits work, I don't know about older ones.